lms/urls.py

```diff
@@ -586,6 +587,11 @@
     urlpatterns += (
         url(r'', include('third_party_auth.urls')),
         url(r'^login_oauth_token/(?P<backend>[^/]+)/$', 'student.views.login_oauth_token'),
+        url(
```

Comment:
Can we create a namespace for this auth endpoint? Also, we should add a version number in the URL.